payjoin-cli/src/db/v2.rs

```diff
@@ -217,4 +218,15 @@ impl Database {
 
         Ok(session_ids)
     }
+
+    pub(crate) fn get_send_session_receiver_pk(
+        &self,
+        session_id: &SessionId,
+    ) -> Result<HpkePublicKey> {
+        let conn = self.get_connection()?;
+        let mut stmt =
+            conn.prepare("SELECT receiver_pubkey FROM send_sessions WHERE session_id = ?1")?;
+        let receiver_pubkey: Vec<u8> = stmt.query_row(params![session_id.0], |row| row.get(0))?;
+        Ok(HpkePublicKey::from_compressed_bytes(&receiver_pubkey).expect("Valid receiver pubkey"))
```

Comment:
I think this is a `Deserialize` Error. Should that `db::Error::Deserialize` variant be a unit type so that it's appropriate here as well?

Or do we just need a new variant? I can see how this wouldn't be the end of the world to panic because there's nothing else to do *as long as get_send_session_receiver_pk is only being used in the one place it currently is*

Reply:
> if its truly undeserializable you got bigger problems

Ack and merging